packages/main/src/TextArea.js

```diff
@@ -329,10 +329,7 @@ const metadata = {
  *
  * <h3 class="comment-api-title">Overview</h3>
  *
- * The <code>ui5-textarea</code> component provides large spaces for text
- * entries in the form of multiple rows.
- * It has the functionality of the <code>TextField</code> with the additional
- * functionality for multiline texts.
+ * The <code>ui5-textarea</code> component is used to enter multiple lines of text.
```

Comment:
Maybe "The <code>ui5-textarea</code> is a multi-line text input component" sound a bit better?